proto/public/uber/cadence/api/v1/history.proto

```diff
@@ -0,0 +1,499 @@
+    WorkflowExecutionTerminatedEventAttributes workflow_execution_terminated_event_attributes = 29;
+    WorkflowExecutionCancelRequestedEventAttributes workflow_execution_cancel_requested_event_attributes = 30;
+    WorkflowExecutionCanceledEventAttributes workflow_execution_canceled_event_attributes = 31;
+    RequestCancelExternalWorkflowExecutionInitiatedEventAttributes request_cancel_external_workflow_execution_initiated_event_attributes = 32;
+    RequestCancelExternalWorkflowExecutionFailedEventAttributes request_cancel_external_workflow_execution_failed_event_attributes = 33;
+    ExternalWorkflowExecutionCancelRequestedEventAttributes external_workflow_execution_cancel_requested_event_attributes = 34;
+    WorkflowExecutionContinuedAsNewEventAttributes workflow_execution_continued_as_new_event_attributes = 35;
+    StartChildWorkflowExecutionInitiatedEventAttributes start_child_workflow_execution_initiated_event_attributes = 36;
+    StartChildWorkflowExecutionFailedEventAttributes start_child_workflow_execution_failed_event_attributes = 37;
+    ChildWorkflowExecutionStartedEventAttributes child_workflow_execution_started_event_attributes = 38;
+    ChildWorkflowExecutionCompletedEventAttributes child_workflow_execution_completed_event_attributes = 39;
+    ChildWorkflowExecutionFailedEventAttributes child_workflow_execution_failed_event_attributes = 40;
+    ChildWorkflowExecutionCanceledEventAttributes child_workflow_execution_canceled_event_attributes = 41;
+    ChildWorkflowExecutionTimedOutEventAttributes child_workflow_execution_timed_out_event_attributes = 42;
+    ChildWorkflowExecutionTerminatedEventAttributes child_workflow_execution_terminated_event_attributes = 43;
+    SignalExternalWorkflowExecutionInitiatedEventAttributes signal_external_workflow_execution_initiated_event_attributes = 44;
+    SignalExternalWorkflowExecutionFailedEventAttributes signal_external_workflow_execution_failed_event_attributes = 45;
+    ExternalWorkflowExecutionSignaledEventAttributes external_workflow_execution_signaled_event_attributes = 46;
+    UpsertWorkflowSearchAttributesEventAttributes upsert_workflow_search_attributes_event_attributes = 47;
+  }
+}
+
+message WorkflowExecutionStartedEventAttributes {
+  WorkflowType workflow_type = 1;
+  string parent_workflow_domain = 2;
+  WorkflowExecution parent_workflow_execution = 3;
+  int64 parent_initiated_event_id = 4;
+  TaskList task_list = 5;
+  Payload input = 6;
+  google.protobuf.Duration execution_start_to_close_timeout = 7 [(gogoproto.stdduration) = true];
+  google.protobuf.Duration task_start_to_close_timeout = 8 [(gogoproto.stdduration) = true];
+  string continued_execution_run_id = 9;
+  ContinueAsNewInitiator initiator = 10;
+  Failure continued_failure = 11;
+  Payload last_completion_result = 12;
+  // This is the runID when the WorkflowExecutionStarted event is written
+  string original_execution_run_id = 13;
```

Comment:
iirc this != current run id when you do a reset?  or is it something else?  may be worth mentioning in a comment.

Reply:
This is for previous runId when doing continue as new. Updated the comment.

Reply:
perfect, thanks!